pkg/lisp/go.sum

```diff
@@ -1,128 +1,275 @@
+4d63.com/gochecknoinits v0.0.0-20200108094044-eb73b47b9fc4/go.mod h1:4o1i5aXtIF5tJFt3UD1knCVmWOXg7fLYdHVu6jeNcnM=
```

Comment:
This file must have gotten changed by accident, and is not part of this PR. This should be reverted.